ha/alert/node_alert_monitor.py

```diff
@@ -27,9 +30,53 @@ def __init__(self):
         Init node alert monitor
         """
         super(NodeAlertMonitor, self).__init__()
+        self.process = SimpleCommand()
+
+    def _get_online_nodes(self):
+        """
+        Get list of online nodes ids.
+        """
+        online_nodes_xml = self.process.run_cmd("crm_mon --as-xml")
```

Comment:
Can we move all such string literals to const.py file in this directory?

Reply:
Done